packages/fxa-auth-server/test/local/routes/devices-and-sessions.js

```diff
@@ -72,8 +73,23 @@ function makeRoutes(options = {}, requireMocks) {
   );
 }
 
-function runTest(route, request, onSuccess, onError) {
-  return route.handler(request).then(onSuccess, onError).catch(onError);
+async function runTest(route, request, onSuccess, onError) {
+  try {
+    const response = await route.handler(request);
+    if (route.options.response.schema) {
+      await Joi.validate(response, route.options.response.schema);
+    }
```

Comment:
This ensures that we run response validation on each test, which is something we do in a handful of other route test suites. There's a nice refactor in here somewhere to extract this into a shared helper, but not for this PR.

Reply:
I imagine we will probably catch a few more of these response validation errors if we pull into a shared helper.